src/05_lists.py

```diff
@@ -7,23 +7,25 @@
 # For the following, DO NOT USE AN ASSIGNMENT (=).
 
 # Change x so that it is [1, 2, 3, 4]
-# YOUR CODE HERE
+x.append(4)
 print(x)
 
 # Using y, change x so that it is [1, 2, 3, 4, 8, 9, 10]
```

Comment:
This works. Check out the extend() method though 👍

Reply:
They have the same run time complexity but using the built end method is a bit more "pythonic"